release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -268,6 +268,57 @@ For more information, see xref:../scalability_and_performance/what-huge-pages-do
 [id="ocp-4-8-monitoring"]
 === Monitoring
 
+[id="ocp-4-8-monitoring-alerting-rule-changes"]
+==== Alerting rule changes
+
+{product-title} 4.8 includes the following alerting rule changes:
+
+.Alerting rule changes
+[%collapsible]
+====
+* The `ThanosSidecarPrometheusDown` alert severity is updated from _critical_ to _warning_.
+* The `ThanosSidecarUnhealthy` alert severity is updated from _critical_ to _warning_.
+* The `ThanosQueryHttpRequestQueryErrorRateHigh` alert severity is updated from _critical_ to _warning_.
+* The `ThanosQueryHttpRequestQueryRangeErrorRateHigh` alert severity is updated from _critical_ to _warning_.
+* The `ThanosQueryInstantLatencyHigh` critical alert is removed. This alert fired if Thanos Querier had a high latency for instant queries.
+* The `ThanosQueryRangeLatencyHigh` critical alert is removed. This alert fired if Thanos Querier had a high latency for range queries.
+* For all Thanos Querier alerts, the `for` duration is increased to 1 hour.
+* For all Thanos sidecar alerts, the `for` duration is increased to 1 hour.
+
+[NOTE]
+=====
+Red Hat does not guarantee backward compatibility for metrics, recording rules, or alerting rules.
+=====
+====
+
+[id="ocp-4-8-monitoring-recording-rules-added-builds-by-strategy"]
+==== Recording rules added for builds by strategy
+
+Recording rules are added that aggregate the number of builds running on a cluster by strategy.
+// CHECK if this note should also include the following specific information: These rules produce a time series with strategy labels of `docker`, `source`, `jenkinspipeline`, and `custom`.
+
+[id="ocp-4-8-monitoring-kube-state-metrics-2.0.0-upgrade"]
+==== kube-state-metrics upgraded to version 2.0.0-rc.1
```

Comment:
OCP 4.8, expect kube-state-metrics, prometheus-operator is bumped to 0.48.1, thanos now is 0.20.2, grafana is 7.5.5, prometheus is 2.26.0, node-exporter is 1.1.2

I'm kindly think we don't need to mention these changes in doc, WDYT? @simonpasquier

Reply:
I think we wanted to mention kube-state-metrics because it's a major version bump and many metrics have changed.

I see that we mentioned a few version bumps in the 4.7 docs (https://docs.openshift.com/container-platform/4.7/release_notes/ocp-4-7-release-notes.html#ocp-4-7-monitoring-stack-dependency-version-updates) but I don't have a strong opinion about whether we should mention everything or only the notable ones.

Reply:
I'll add the bumps that Junqi mentions above. Please let me know if I should remove any/if any are not notable. 👍  Per previous conversations, I think we decided that it's helpful for folks to know what component versions are being used in the stack.

Reply:
@lbarbeevargas FYI: prometheus will updated to 2.26.1 later for 4.8